assets/model_monitoring/components/src/shared_utilities/span_tree_utils.py

```diff
@@ -192,13 +201,19 @@ class SpanTree:
     def __init__(self, spans: List[SpanTreeNode]) -> None:
         """Spantree constructor to build up tree from span list."""
         self._span_node_map: Dict[str, SpanTreeNode] = {}
+        self._possible_root_spans = []
```

Comment:
I suppose the only advantage is for keeping the root_span property logic simple but I will look into changing the code as you mentioned in a follow up PR since some of Vivienne's e2e testing is blocked until this PR is merged.